pyproject.toml

```diff
@@ -27,10 +27,11 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-nautobot = "^1.4.0"
+nautobot = "^1.5.13"
```

Comment:
Nice learning from another similar MR - we don't have to change this. If the version is smaller than 1.5.13 it just doesn't render any metrics - if the version is bigger then or equal to it renders them. This should maybe go in the docs instead?